helper_test.go

```diff
@@ -342,19 +342,18 @@ func createPlannedRun(t *testing.T, client *Client, w *Workspace) (*Run, func())
 			t.Fatal(err)
 		}
 
-		if r.Status == RunPlanned || r.Status == RunPolicyChecked || r.Status == RunPolicyOverride {
-			break
+		switch r.Status {
+		case RunPlanned, RunCostEstimated, RunPolicyChecked, RunPolicyOverride:
+			return r, rCleanup
 		}
 
-		if i > 30 {
+		if i > 45 {
```

Comment:
support an extra run stage. it got too close to the full 30s during my local testing. 45 feels safer, depending on the speed of your laptop w/ containers.